hw_01/part_1/js/script.js

```diff
@@ -0,0 +1,50 @@
+//NUMBERS
+
+alert(Math.PI.toFixed(2)); //"3.14"
+
+var num = 0.6 + 0.7;
+if (num == 1.3) {
```

Comment:
Try to avoid usage `==` when you compare something. `==` can give you unexpected result. Always use `===` instead. Just [read this short article](https://blog.webdevsimplified.com/2020-08/==-vs-===/) and you will understand why :)
[More information about compare operators](https://javascript.info/comparison)